.github/workflows/code-quality.yaml

```
@@ -21,50 +21,38 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php-version: [ '8.2', '8.3', '8.4']
-                dependency-version: ['']
-                symfony-version: ['']
-                minimum-stability: ['stable']
-                include:
-                    # lowest deps
-                    -   php-version: '8.2'
-                        dependency-version: 'lowest'
-                    # LTS version of Symfony
-                    -   php-version: '8.2'
-                        symfony-version: '6.4.*'
+                php-version: [ '8.4' ]
         steps:
-            -   name: Checkout
-                uses: actions/checkout@v4
+            - name: Checkout
+              uses: actions/checkout@v4
 
             - name: Configure environment
               run: |
                   echo COLUMNS=120 >> $GITHUB_ENV
-                  echo COMPOSER_MIN_STAB='composer config minimum-stability ${{ matrix.minimum-stability || 'stable' }} --ansi' >> $GITHUB_ENV
-                  echo COMPOSER_UP='composer update ${{ matrix.dependency-version == 'lowest' && '--prefer-lowest' || '' }} --no-progress --no-interaction --ansi --ignore-platform-req=ext-mongodb' >> $GITHUB_ENV
+                  echo COMPOSER_UP='composer update --no-progress --no-interaction --ansi --ignore-platform-req=ext-mongodb' >> $GITHUB_ENV
                   echo PHPSTAN='vendor/bin/phpstan' >> $GITHUB_ENV
 
                   PACKAGES=$(find src/ -mindepth 2 -type f -name composer.json -not -path "*/vendor/*" -printf '%h\n' | sed 's/^src\///' | grep -Ev "examples" | sort |  tr '\n' ' ')
                   echo "Packages: $PACKAGES"
                   echo "PACKAGES=$PACKAGES" >> $GITHUB_ENV
 
-            -   name: Setup PHP
-                uses: shivammathur/setup-php@v2
-                with:
-                    php-version: '8.2'
-                    tools: flex
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: '8.2'
```

```suggestion
                  php-version: '${{ matrix.php-version }}'
```
? 
Or maybe we can get rid of the following part ?

```
matrix:
    php-version: [ '8.4' ]
```